de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/exception/BankingException.java

```diff
@@ -95,8 +95,9 @@ protected ITranslatableString buildMessage()
 		if (targetAcct != null)
 		{
 			sb.append("Zielkonto:\n");
-			sb.append("\tEmpfaenger: ")
-					.append(targetAcct.getC_BPartner().getName());
+			 // TODO tbp: how to fix this?
```

Comment:
how to fix what in particular; anyways, this hardcoded stuff needs fixing, sooner or later. an additional unsolved TODO is therefore not a big deal for me

Reply:
_how to_ referred to getting the bpartner name. there's no more `C_BP_BankAccount.getC_BPartner()`, only `C_BP_BankAccount.getC_BPartner_ID()`, and i dont think loading the bpdao service and reading the name each time is a good idea.

Reply:
Generally, i think it's not a terrible idea bc the record is cached if you load it outside of any particular transaction, which the DAO-methods usually do.
And besides: make an educated guess as to what `C_BP_BankAccount.getC_BPartner()` did under the hood 🤓